pkg/manager/internal/controllers/organizationroletemplate_controller.go

```diff
@@ -193,6 +229,41 @@ func (r *OrganizationRoleTemplateReconciler) reconcileRBACForOrganization(ctx co
 	return nil
 }
 
+func (r *OrganizationRoleTemplateReconciler) reconcileRBACForProject(ctx context.Context, organizationRoleTemplate *corev1alpha1.OrganizationRoleTemplate, organization *storagev1alpha1.Organization, project *storagev1alpha1.Project) error {
+	// Reconcile Role.
+	role := &rbacv1.Role{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      organizationRoleTemplate.Name,
+			Namespace: project.Status.Namespace.Name,
+		},
+		Rules: organizationRoleTemplate.Spec.Rules,
+	}
+	if err := r.reconcileRole(ctx, role, organizationRoleTemplate); err != nil {
+		return err
+	}
+
+	// Reconcile RoleBindings.
+	if organizationRoleTemplate.HasBinding(corev1alpha1.BindToOwners) {
+		roleBinding := &rbacv1.RoleBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      organizationRoleTemplate.Name,
+				Namespace: project.Status.Namespace.Name,
+			},
+			// Here we are creating RoleBindings for Organization Owner, not Project Owner, since OrganizationRoleTemplate is used to config permissions of Organization Owners.
+			Subjects: organization.Spec.Owners,
+			RoleRef: rbacv1.RoleRef{
+				APIGroup: "rbac.authorization.k8s.io",
```

Comment:
IIRC there's a constant for it in the `rbacv1` package

Reply:
fixed, I am using `rbacv1.GroupName`